src/Parser/Shaman/Restoration/Modules/Abilities.js

```diff
@@ -13,7 +12,7 @@ class Abilities extends CoreAbilities {
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
         charges: 2,
```

Comment:
instead of having 2 entries for riptide you could set the number of charges based on the enabled logic (like you do with healing stream below)